overlord/ifacestate/ifacestate_test.go

```diff
@@ -586,7 +608,7 @@ func (s *interfaceManagerSuite) TestAutoconnectNoConflictOnConnect(c *C) {
 	s.state.Unlock()
 
 	err := s.createAutoconnectChange(c, task)
-	c.Assert(err, IsNil)
+	c.Assert(err, ErrorMatches, `task should be retried`)
```

Comment:
Is the test name (TestAutoconnectNoConflictOnConnect) still accurate?

Reply:
Fixed, thanks for noticing.